modules/agar/.eslintrc.json

```diff
@@ -1,5 +1,6 @@
 {
   "rules": {
+    "@tinymce/no-unimported-promise": "off",
```

Comment:
This isn't needed here as it's in the root file instead. These files only exist as overrides to the root file.